common/src/main/java/com/sequenceiq/cloudbreak/jerseyclient/AugmentedRestClient.java

```diff
@@ -7,5 +7,5 @@
 
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
-public @interface RetryingRestClient {
+public @interface AugmentedRestClient {
```

Comment:
What was the problem with RetryingRestClient? This is retrying only like previously, right?

Reply:
'augmented' is a bit too general

Reply:
@topolyai5, no, please check the aspect's code.

Reply:
@bbihari any suggestions?

Reply:
MetricCollectingAndRetryingRestClient would be better? I think the current name is fine.